src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -122,12 +123,14 @@ namespace dpdk {
           funcs.pop_front();
           f();
           cond.notify_all();
+          if (exit_thread) {
+            pthread_exit(NULL);
```

Comment:
no need to call `pthread_exit()`. as `std::thread` is able to take care of this.

Reply:
ok